pkg/plugins/resources/k8s/native/k4k8s

```diff
@@ -0,0 +1,1331 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
What's this file all about?

Reply:
arg, that found it's way in from the KIC PR